### PR TITLE
Spray logging improvement

### PR DESCRIPTION
**Pull request by TDSSS**
## What Does This PR Do
This PR does two things.
1. Logs when you use a fire extinguisher at a dangerous temperature
2. Brings spray bottle logging in line with the new, improved log viewer so it gets shown in logging viewer

## Why It's Good For The Game
Scalding hot liquids are very dangerous for random civies without masks or hats and can be gotten with just a lighter and a liquid container. They should be properly logged so we can ban people that use them to murderbone.

## Changelog
:cl:
tweak: minor logging improvement to sprays
/:cl:

### CODE REVIEW

**Comment by TDSSS** [comment]
How tf did that merge conflict in the first place, lol.

File: code/game/objects/items/weapons/extinguisher.dm [hunk 1]
```diff
@@ -106,6 +106,8 @@
 
 		src.last_use = world.time
 
+		if(reagents.chem_temp > 300 || reagents.chem_temp < 280)
+			add_attack_logs(user, null, "Sprayed with superheated or cooled fire extinguisher at Temperature [reagents.chem_temp]K")
```
Comment (farie82) [inline code/game/objects/items/weapons/extinguisher.dm:110]: Can include the target here to see which direction they sprayed in in the logs

File: code/modules/reagents/reagent_containers/spray.dm [hunk 2]
```diff
@@ -57,8 +57,8 @@
 	var/attack_log_type = ATKLOG_MOST
 	if(reagents.has_reagent("sacid") || reagents.has_reagent("facid") || reagents.has_reagent("lube"))
 		attack_log_type = ATKLOG_FEW
-	add_attack_logs(user, A, "sprayed [contents_log] at [reagents.chem_temp]K using [src]", attack_log_type)
-	log_game("[key_name(user)] used a spray bottle at [COORD(user)] - Contents: [contents_log] - Temperature: [reagents.chem_temp]K")
+
+	add_attack_logs(user, null, "Used a potentially harmful spray bottle. Contents: [contents_log] - Temperature: [reagents.chem_temp]K", attack_log_type)
```
Comment (farie82) [inline code/modules/reagents/reagent_containers/spray.dm:61]: Same here. Especially for the space cleaner spray it's important to know what target was being sprayed on

**Review by farie82 — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/extinguisher.dm (hunk 1 above)
Comment (TDSSS) [reply code/game/objects/items/weapons/extinguisher.dm]: done

**Review by farie82 — APPROVED** [review]

**Review by variableundefined — APPROVED** [review]

**Pull request merged**